common/changes/@microsoft/api-extractor/pgonzal-dts-generator-5_2018-01-12-01-56.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor",
+      "comment": "Add some incremental improvements for the experimental PackageTypingsGenerator feature",
```

Comment:
>some [](start = 22, length = 4)

do you want to elaborate?

Reply:
This feature is unreleased

---
In reply to: [161125673](https://github.com/Microsoft/web-build-tools/pull/483#discussion_r161125673) [](ancestors = 161125673)